11-payment-encoding.md

```diff
@@ -202,6 +203,8 @@ A reader:
   description.
   - if a valid `n` field is provided:
     - MUST use the `n` field to validate the signature instead of performing signature recovery.
+  - if there is a valid `s` field:
```

Comment:
gate with `basic_mpp` feature bit?

Reply:
It is gated on the writer side.  But what's a reader to do if the `s` is set and mpp isn't, or vice versa?  I chose to keep it simple: if `s` is there, use it.

Reply:
sgtm!